MAUI/Polar-Charts/Axis/Ticklines.md

```diff
@@ -49,6 +49,43 @@ chart.SecondaryAxis = secondaryAxis;
 
 {% endtabs %}
 
+## Positioning the ticklines
+
+The positioning of tick lines inside or outside the chart area can be controlled using the [TickPosition]() property. By default, the tick lines are positioned `AxisElementPosition.Outside` the chart area.
+
+N> Polar chart supports TickPosition for secondary axis only.
+
```

Comment:
This is only applicable to the secondary axis of the polar chart.

Reply:
Updated the content